lib/ansible/modules/net_tools/ldap/_ldap_attr.py

```diff
@@ -36,6 +36,10 @@
     possible to see spurious changes when target and actual values are
     semantically identical but lexically distinct.
 version_added: '2.3'
+deprecated:
+  removed_in: '2.14'
+  why: 'The current "ldap_attr" module does not support ldap attribute insertations or deletions with objectClass dependencies..'
+  alternative: 'Use M(ldap_attrs) instead. Deprecated in 2.10. Use M(ldap_attrs) instead.'
```

Comment:
```suggestion
  alternative: 'Use M(ldap_attrs) instead. Deprecated in 2.10.'
```
Mentioning `ldap_attrs` once is enough :)